### PR TITLE
docs cookbook polish

### PR DESCRIPTION
**Pull request by alexcg1**
Just some minor wording cleanup. Re-reading and editing also helps it live in my brain

- docs(cookbook): polish english
- docs(cookbook): polish english

### CODE REVIEW

**Review by hanxiao — CHANGES_REQUESTED** [review]
- text in the code should not be changed, plz do not batch replace all `document` to `Document`, especially when they are in the code
- no need to change toc

File: .github/2.0/cookbooks/CleanCode.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Cookbook on Clean Code
```
Comment (theUnkownName) [inline .github/2.0/cookbooks/CleanCode.md:1]: Shouldn't we follow the same guidelines as for other READMES? So title case here too?

File: .github/2.0/cookbooks/Document.md [hunk 2]
```diff
@@ -574,12 +574,12 @@ DocumentArray has 3 items:
 
 ### Use `itertools` on `DocumentArray`
 
-As `DocumenArray` is an `Iterable`, you can also use [Python built-in `itertools` module](https://docs.python.org/3/library/itertools.html) on it. This enables advanced "iterator algebra" on the `DocumentArray`.
+As `DocumentArray` is an `Iterable`. You can also use [Python's built-in `itertools` module](https://docs.python.org/3/library/itertools.html) on it. This enables advanced "iterator algebra" on the `DocumentArray`.
```
Comment (theUnkownName) [inline .github/2.0/cookbooks/Document.md:577]: We really should have a dot here? those two sentences are connected right?

Comment (alexcg1) [reply .github/2.0/cookbooks/Document.md:577]: Ah, my bad. I'll fix

File: .github/2.0/cookbooks/CleanCode.md (hunk 1 above)
Comment (alexcg1) [reply .github/2.0/cookbooks/CleanCode.md:1]: Title case doesn't mean every word capitalized :) This is in title case already

[Minor words shouldn't be capitalized](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case)

File: .github/2.0/cookbooks/Document.md (hunk 2 above)
Comment (alexcg1) [reply .github/2.0/cookbooks/Document.md:577]: Fixed

**Comment by alexcg1** [comment]
> * text in the code should not be changed, plz do not batch replace all `document` to `Document`, especially when they are in the code

Fixed

**Comment by alexcg1** [comment]
@hanxiao @theUnkownName please let me know if you find any other glitches. Git was throwing merge conflict stuff at me earlier - I'm pretty sure all of my edits and corrections got through (and I checked), but something might've fallen through the gaps

**Comment by theUnkownName** [comment]
> @hanxiao @theUnkownName please let me know if you find any other glitches. Git was throwing merge conflict stuff at me earlier - I'm pretty sure all of my edits and corrections got through (and I checked), but something might've fallen through the gaps

Is it me maybe? I'm not seeing any of the changes

**Comment by alexcg1** [comment]
For some reason we can't see any of the changes on the GitHub interface. But if you go to the branch in the repo you can see the updated docs (either [here](https://github.com/jina-ai/jina/tree/docs-cookbook-polish/.github/2.0/cookbooks) or in your terminal)

File: .github/2.0/cookbooks/Document.md [hunk 3]
```diff
@@ -96,25 +96,25 @@ d4 = Document(content='https://static.jina.ai/logo/core/notext/light/logo.png')
...
 
 ```text
-<jina.types.document.Document id=2ca74b98-aed9-11eb-b791-1e008a366d48 mimeType=text/plain text=hello at 6247702096>
-<jina.types.document.Document id=2ca74f1c-aed9-11eb-b791-1e008a366d48 buffer=DDE= mimeType=text/plain at 6247702160>
-<jina.types.document.Document id=2caab594-aed9-11eb-b791-1e008a366d48 blob={'dense': {'buffer': 'AQAAAAAAAAACAAAAAAAAAAMAAAAAAAAA', 'shape': [3], 'dtype': '<i8'}} at 6247702416>
-<jina.types.document.Document id=4c008c40-af9f-11eb-bb84-1e008a366d49 uri=https://static.jina.ai/logo/core/notext/light/logo.png mimeType=image/png at 6252395600>
+<jina.types.Document.Document id=2ca74b98-aed9-11eb-b791-1e008a366d48 mimeType=text/plain text=hello at 6247702096>
+<jina.types.Document.Document id=2ca74f1c-aed9-11eb-b791-1e008a366d48 buffer=DDE= mimeType=text/plain at 6247702160>
+<jina.types.Document.Document id=2caab594-aed9-11eb-b791-1e008a366d48 blob={'dense': {'buffer': 'AQAAAAAAAAACAAAAAAAAAAMAAAAAAAAA', 'shape': [3], 'dtype': '<i8'}} at 6247702416>
+<jina.types.Document.Document id=4c008c40-af9f-11eb-bb84-1e008a366d49 uri=https://static.jina.ai/logo/core/notext/light/logo.png mimeType=image/png at 6252395600>
```
Comment (hanxiao) [inline .github/2.0/cookbooks/Document.md:102]:
```suggestion
<jina.types.document.Document id=2ca74b98-aed9-11eb-b791-1e008a366d48 mimeType=text/plain text=hello at 6247702096>
<jina.types.document.Document id=2ca74f1c-aed9-11eb-b791-1e008a366d48 buffer=DDE= mimeType=text/plain at 6247702160>
<jina.types.document.Document id=2caab594-aed9-11eb-b791-1e008a366d48 blob={'dense': {'buffer': 'AQAAAAAAAAACAAAAAAAAAAMAAAAAAAAA', 'shape': [3], 'dtype': '<i8'}} at 6247702416>
<jina.types.document.Document id=4c008c40-af9f-11eb-bb84-1e008a366d49 uri=https://static.jina.ai/logo/core/notext/light/logo.png mimeType=image/png at 6252395600>
```

Comment (hanxiao) [reply .github/2.0/cookbooks/Document.md:102]: same for all incode `document`, they have to be lower case

Comment (alexcg1) [reply .github/2.0/cookbooks/Document.md:102]: I changed those all already. For some reason my commit isn't showing in the PR, but it's showing in the branch itself. God knows how or why

Comment (alexcg1) [reply .github/2.0/cookbooks/Document.md:102]: WTF? I'm going to look into this. I DEFINITELY changed all of those yesterday. I remember going through it by hand. And now it's not showing on my local box

Comment (alexcg1) [reply .github/2.0/cookbooks/Document.md]: OK, fixed. And I ran a grep on all the files to ensure there are no remaining occurrences

**Pull request merged**